src/main/java/com/google/graphgeckos/dashboard/storage/DataRepository.java

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.storage;
+
+/**
+ * An interface for providing unified I/O to the storage component.
```

Comment:
JavaDoc should describe the semantics of your components, not repeat the syntax. The code already says that this is an interface. No need to repeat that. And the fact that it provides an unified API follows from it being an interface as well. The only interesting part here is "storage component". That however could benefit from a bit more meat. 

Good JavaDoc would be something like "A storage component for revision data." And then additional helpful context. 

(I don't know what we're developing here, so this is generic. You should have more specifics).

Reply:
Thank you for the input! I addressed this with [the latest commit](https://github.com/googleinterns/step240-2020/pull/23/commits/f99d1cc5c7363aefdd71ba2ca19864ad0a77ddcf). Please let me know if the new JavaDoc makes more sense.